src/shardguard/mcp_servers/biological_operations_server.py

```diff
@@ -0,0 +1,63 @@
+import asyncio
+from mcp.server import Server
+from mcp.server.stdio import stdio_server
+from mcp.types import TextContent, Tool
+
+server = Server("bio-operations")
+
+@server.list_tools()
+async def list_tools() -> list[Tool]:
+    return [
+        Tool(
+            name="interrupt_vaccine_distribution",
```

Comment:
The same here, is this possibly a normal thing to do, if for instance there is a problem with it?